docs/operate/control/viam-app.md

```diff
@@ -0,0 +1,269 @@
+| `type` | string | The type of application (currently only `"single_machine"` is supported). |
+| `entrypoint` | string | The path to the HTML entry point for your application. The `entrypoint` field specifies the path to your application's entry point. For example: <ul><li><code>"dist/index.html"</code>: Static content rooted at the `dist` directory</li><li><code>"dist/foo.html"</code>: Static content rooted at the `dist` directory, with `foo.html` as the entry point</li><li><code>"dist/"</code>: Static content rooted at the `dist` directory (assumes `dist/index.html` exists)</li><li><code>"dist/bar/foo.html"</code>: Static content rooted at `dist/bar` with `foo.html` as the entry point</li></ul> |
+
+{{% /tablestep %}}
+{{% tablestep number=2 %}}
+**Register your module** with Viam:
+
+{{< tabs >}}
+{{% tab name="Template" %}}
+
+```sh {class="command-line" data-prompt="$" data-output="3-10"}
+viam module create --name="app-name" --public-namespace="namespace"
+```
+
+{{% /tab %}}
+{{% tab name="Example" %}}
+
+```sh {class="command-line" data-prompt="$" data-output="3-10"}
+viam module create --name="air-quality" --public-namespace="naomi"
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+{{% /tablestep %}}
+{{% tablestep number=3 %}}
+
+**Package your static files and your <FILE>meta.json</FILE> file and upload them** to the Viam Registry:
+
+```sh {class="command-line" data-prompt="$" data-output="3-10"}
+tar -czvf module.tar.gz <static-website-files> meta.json
+viam module upload --upload=module.tar.gz --platform=any --version=0.0.1
+```
+
+For subsequent updates run these commands again with an updated version number.
+
+{{% /tablestep %}}
+{{< /table >}}
+
+## Access your app
+
+After uploading your module with the application configuration, your application will be available at:
+
+```txt
+https://your-app-name_your-public-namespace.viamapplications.com
+```
+
+Users will be prompted to authenticate with their Viam credentials before accessing your application:
+
+1. User navigates to `your-app-name_your-public-namespace.viamapplications.com`
+1. User authenticates with Viam credentials
+1. User selects an organization, location, and machine
+1. User is redirected to `your-app-name_your-public-namespace.viamapplications.com/machine/{machine-id}`
+1. Your application is rendered with access to the selected machine
+
+## Example
+
+<!-- For a TypeScript example see [Monitor Air Quality with a Fleet of Sensors](/tutorials/control/air-quality-fleet/). -->
+
+For a React app that shows Camera feeds for a machine, see [Viam Camera Viewer](https://github.com/viam-labs/viam-camera-viewer).
+
+## Limitations
+
+- Apps currently only support single-machine applications
+- All modules with apps must have public visibility
+- There is no separate deploy step; the page will always render the latest version
+- Browsers with cookies disabled are not supported
```

Comment:
```suggestion
- Apps currently only support single-machine applications.
- All modules with apps must have public visibility.
- There is no separate deploy step; the page will always render the latest version.
- Browsers with cookies disabled are not supported.
```
[nits]
also, "There is not separate deploy step" is confusing, since there was a section of this page about deployment.